ccpp/data/GFS_typedefs.F90

```diff
@@ -3419,6 +3423,10 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     real(kind=kind_phys) :: bl_mynn_closure   = 2.6                   !<   <= 2.5  only prognose tke
                                                                       !<   2.5 < and < 3.0, prognose tke and q'2
                                                                       !<   >= 3.0, prognose tke, q'2, T'2, and T'q'
+    logical              :: sfclay_compute_diag = .false.
+    logical              :: sfclay_compute_flux = .false.
```

Comment:
This differs from the original defaults. I'm testing sfclay_compute_flux=.true. right now to see if it fixes the problem.

Reply:
I've run a bunch of rrfs and rap regression tests with `sfclay_compute_flux=.true.` and they're passing. It seems this is the problem.

Reply:
So, is the solution to:
1) add  sfclay_compute_flux=.true. to the HRRR/RRFS namelists
2) keep the non-HRRR/RRFS namelist as is (default to .false.). Also Keep RAP namelists as is.
3) run new baselines for the HRRR/RRFS namelists
4) then complete the regression testing
5) merge
?

Reply:
For (2), we also need to configure nsst to turn off.

Reply:
I understand step 2. Are you talking about the workflow?

Reply:
I think so, assuming it's the workflow that determines which namelists are used for each test/suite. My thinking is that we should probably create a regresion test that actually mimics the way RRFS is run (with sfclay_compute_flux=.true.). This would require a new baseline. The rest of the regression tests can stay with the default setting (sfclay_compute_flux=.false.) but the tests that use the mynn surface layer scheme and also use nsst will require new baselines since the original regression tests (with sfclay_compute_flux hard-coded as .false.) were overwritten with the baselines generated from Mike's PR.